config/public-testnet.yaml

```diff
@@ -131,21 +131,19 @@ checks:
   pt-testnet-load:
     options:
       content-size: 50000000
-      postage-amount: 1000000000 # 1024/15*60*60*24*7
+      postage-amount: 1428084000 # (28335/12)*60*60*24*7
```

Comment:
hopefully we can drop this amount soon 😆

Reply:
working on it :)